_config.yml

```diff
@@ -157,19 +158,16 @@ optional_front_matter:
 # Exclude all .md directories and files except page-builder/ to prevent painfully long build times.
 
 relative_links:
+  enabled:     true
+  collections: false
   exclude:
-    - _includes
```

Comment:
I don't think we need to crawl the `_includes` directory for relative links. I think removing it from the exclude list will increase build time.

Reply:
> I don't think we need to crawl the _includes directory for relative links.

I don't think it's been crawled, considering the Jekyll architecture. The files in __includes_ do not contain publishable source files, only chunks of content that's been included in future topic at one of processing layers. I assume that it's been crawled at that layer, when the content was included.

> will increase build time.

I checked this, it didn't change the build time.